server/lib/models/model.go

```diff
@@ -3,55 +3,50 @@ package models
 import (
 	"log"
 	"fmt"
+	"strings"
 	"database/sql"
 
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/utils"
 
-	_ "github.com/cockroachdb/cockroach/sql/driver"
+	_ "github.com/cockroachdb/pq"
 	"github.com/umegaya/gorp"
+	"github.com/umegaya/yue"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/yue (can't find import: github.com/umegaya/yue)